Connects to #282. Connects to #284. Data Hub native graphical cluster reports and full PASS1B-06 phenotype table.

jimmyzhen opened the pull request:
### Key Changes:

1. Converted the large external HTML graphical clustering reports (`/graphical-clustering`) into something more native to the Data Hub portal with a more integrated look and feel. No longer using `iframe` to import these HTML files.
    * All expandable/collapsible code blocks have been removed.
    * All interactive pathway enrichment network graphs have been removed.
    * Because all the graphical clustering reports share the same “Introduction” summary, the text has been removed to minimize repetition and rather shown as part of the main page content.
    * Relevant to the mitochondria paper ONLY - the graphical clustering reports are not available for the cortex, hippocampus, hypothalamus and plasma tissue. This is because there are no clustering graphs/plots in their raw HTML reports.
2. Added new route and component to display the full PASS1B-06 phenotype data table for the phenotype paper submitted at Function.
    * The `*.xlsx` file containing 4 sheets was converted to Google Sheet. The Google Sheet then was published as non-editable page being loaded into an `iframe` on Data Hub.
    * The original `*.xlsx` file can also be downloaded from the page rendered by the component.
4. Other UI changes:
    * On the Project Overview page (`/project-overview`), there are now links to animal/human protocols hosted on the main motrpac.org portal. A “Explore data” button has also been added, linking to the differential abundance search page.
    * On the Methods page (`/methods`), the links to the animal/human protocols have been added to the page summary.

cteng585 reviewed (dismissed):
Everything working as expected. Graphical clustering feels much better to explore with the changes to how the reports are implemented; I feel that I'm experiencing faster loading times for the page as a whole.

Agreed with Mihir's feedback on the Landscape/Mitochondria toggle. I think my other piece of feedback is that it doesn't feel like "Mitochondria-Related Features Only" needs to be in all-caps. Given that it's a pretty big chunk of text already, making it in uppercase comes across as not trusting the user to understand what the toggle did.

jimmyzhen commented:
@cteng585 Thanks for the comments.

I've implemented a solution to address the TOC issue described in Mihir's comment, as well as your comment about the ALL CAP title.

I also implemented the additional full phenotype table as mentioned last Friday.
